sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingAggregationSuite.scala

```diff
@@ -335,4 +344,67 @@ class StreamingAggregationSuite extends StreamTest with BeforeAndAfterAll {
       CheckLastBatch((90L, 1), (100L, 1), (105L, 1))
     )
   }
+
```

Comment:
The idea of this test is injecting a mock state store, which:
- throw an error in its `commit()` method
- mark the `aborted` flag in its `abort()` method
And at the end of the test, we would check if `aborted` is true